file_paths.md

```diff
@@ -121,6 +124,10 @@ when installing puppet-agent 32-bit on 64-bit windows, the
 installation path defaults to `C:\Program Files (x86)\Puppet
 Labs`. The examples below assume 2008/2012 and puppet-agent (64-bit).
 
+The package will create two services `puppet` and `mcollective`
+running as `LocalSystem` by default. It will not create a `puppet`
+user or group.
+
```

Comment:
For PE, it used to be `pe-puppet` for both user/group and service name, and `pe-mcollective` for the mco service. For FOSS, the puppet user/group used to be created by the puppet package and the puppet user was used in the puppet-server and puppetmaster packages/services.